rpc/src/client/event_listener.rs

```diff
@@ -1,216 +0,0 @@
-//! Tendermint Websocket event listener client
```

Comment:
Maybe we should bring this back for now so folks can upgrade their websocket code at their own pace?

Reply:
The trouble with bringing this back is that the types exposed by the event listener conflict with the types I've added in `rpc/src/event.rs`. As far as I can see in IBC it's just this code that needs to be updated: https://github.com/informalsystems/ibc-rs/blob/master/relayer/src/event_monitor.rs

Reply:
Ok fair enough. Let's merge this as is and once ibc is ready to upgrade to our master we can work on integration there.